apis/v1alpha3/zz_generated.conversion.go

```diff
@@ -859,15 +859,11 @@ func autoConvert_v1beta1_Topology_To_v1alpha3_Topology(in *v1beta1.Topology, out
 	out.ComputeCluster = (*string)(unsafe.Pointer(in.ComputeCluster))
 	out.Hosts = (*FailureDomainHosts)(unsafe.Pointer(in.Hosts))
 	out.Networks = *(*[]string)(unsafe.Pointer(&in.Networks))
+	// WARNING: in.NetworkConfigurations requires manual conversion: does not exist in peer-type
```

Comment:
We need to do the manual conversion.

In both `vspherefailuredomain_conversion.go` functions we have to do things similar to:

For the ConvertTo function:

https://github.com/Nutrymaco/cluster-api-provider-vsphere/blob/fd-new-api/apis/v1alpha4/vspheremachine_conversion.go#L33-L48

For the ConvertFrom function:

https://github.com/Nutrymaco/cluster-api-provider-vsphere/blob/fd-new-api/apis/v1alpha4/vspheremachine_conversion.go#L61-L64